Example/test/index.js

```diff
@@ -92,6 +94,14 @@ export default class Example extends Component {
     // const transX = new Value(0);
     const clock = new Clock();
     // const twenty = new Value(20);
+    const a = param("a");
```

Comment:
Do we need these names? why?

Reply:
No we don't need the names - it was just to make it a little more easy to debug.. ;)